spec/02-integration/04-admin_api/02-apis_routes_spec.lua

```diff
@@ -563,6 +563,31 @@ describe("Admin API " .. kong_config.database, function()
             assert.same(json, in_db)
           end
         end)
+        it_content_types("removes optional field with ngx.null", function(content_type)
+          return function()
+            -- TODO: how should ngx.null work with application/www-form-urlencoded?
```

Comment:
It is probably out of the scope of this PR, which is just to make the Admin API/DAO accept the JSON `null` type (which of course does not exist in `application/www-form-urlencoded`), so nothing to worry about imo :)